### ISSUE TITLE
There is a problem to associate Assets

### ISSUE THREAD

**Issue by miguelanruizimagu** [pluginsGLPI/formcreator#2348]
**Describe the bug**
When trying to associate an asset from a question in the Target field to a ticket, it does not appear as selectable.

**To Reproduce**
1. Go to any form that you need. It must have a question with any of assets as GLPI Object type.
![image](https://user-images.githubusercontent.com/88452851/128386785-c0cacb6b-c772-4e4d-a14d-d73715ea2c87.png)

2. Click on Targets field, and go inside.
![image](https://user-images.githubusercontent.com/88452851/128383383-375a51cd-fe72-41af-bf00-416fd7ae9c91.png)

3. Scroll down to Associated elements.
4. Put Equals to answer.
5. There is not a selectable question.
![image](https://user-images.githubusercontent.com/88452851/128386429-d7c82fdb-90e9-415d-9493-d8694478dc98.png)

No options.

**Expected behavior**
You should relate a selectable question:
![image](https://user-images.githubusercontent.com/88452851/128383743-0c3d0984-eeb3-4b28-8797-a22d5a29932b.png)

**Comment by miguelanruizimagu** [comment]
I have noticed that by changing "value" attribute of question, removing the JSON format, it works.
![image](https://user-images.githubusercontent.com/88452851/128387202-a11f46cf-c988-4f40-8636-47a8230e442a.png)

Changing: {"itemtype":"Peripheral"}  -- > "Peripheral"

Then you can see as a result:
![image](https://user-images.githubusercontent.com/88452851/128387374-e200ac3c-1d5b-4d56-aaa9-054222b554f9.png)

**Comment by btry** [comment]
Cehck if your profile settings have devices in aassociatable items to ticlets.

![image](https://user-images.githubusercontent.com/14139801/128388449-1ecb9d97-b734-431b-8158-10fd5264c8c9.png)

![image](https://user-images.githubusercontent.com/14139801/128388479-6fbca9e6-4e83-4a42-b821-f5c4250489c6.png)

**Comment by miguelanruizimagu** [comment]
Right i got it
![image](https://user-images.githubusercontent.com/88452851/128389387-11b7dda9-d946-4c06-8abe-9532b7648956.png)



> I have noticed that by changing "value" attribute of question, removing the JSON format, it works.
> ![image](https://user-images.githubusercontent.com/88452851/128387202-a11f46cf-c988-4f40-8636-47a8230e442a.png)
> 
> Changing: {"itemtype":"Peripheral"} -- > "Peripheral"
> 
> Then you can see as a result:
> ![image](https://user-images.githubusercontent.com/88452851/128387374-e200ac3c-1d5b-4d56-aaa9-054222b554f9.png)
> 
> All of this over FormsCreator 2.12.0

I think is something related with format, because in previous versions is simple like "Peripheral", so without JSON.

**Comment by btry** [comment]
Yes, there was also a problem related to the JSON format. Add this fix https://github.com/pluginsGLPI/formcreator/pull/2315/files

**Comment by btry** [comment]
Hi

Hi

Did you tried the patch #2315 ?
